dirsrvtests/tests/suites/plugins/alias_entries_test.py

```diff
@@ -0,0 +1,93 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2023 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import logging
+import pytest
+import os
+import ldap
+import time
+from lib389.utils import ensure_str
+from lib389.plugins import AliasEntriesPlugin
+from lib389.idm.user import UserAccount
+from lib389._constants import DEFAULT_SUFFIX
+from lib389.topologies import topology_st as topo
+
+log = logging.getLogger(__name__)
+
+TEST_ENTRY_NAME = "entry"
+TEST_ENTRY_DN = "cn=entry," + DEFAULT_SUFFIX
+TEST_ALIAS_NAME = "alias entry"
+TEST_ALIAS_DN = "cn=alias_entry," + DEFAULT_SUFFIX
+EXPECTED_UIDNUM = "1000"
+
+
+def test_entry_alias(topo):
+    """Test that deref search for alias entry works
+
+    :id: 454e85af-0e20-4a36-9b3a-02562b1db53d
+    :setup: Standalone Instance
+    :steps:
+        1. Enable alias entry plugin
+        2. Create entry and alias entry
+        2. Set deref option and do a base search
```

Comment:
numbering is incorrect.

Reply:
> numbering is incorrect.

I'll fix this, and I did run this PR through ASAN and did not find any issues.